api/client/trust.go

```diff
@@ -248,6 +252,8 @@ func notaryError(err error) error {
 		return fmt.Errorf("remote repository out-of-date: %v", err)
 	case trustmanager.ErrKeyNotFound:
 		return fmt.Errorf("signing keys not found: %v", err)
+	case *net.OpError:
```

Comment:
a search for `notaryError` on docker/docker only brings up this result. What am I missing?